spt/OrangeBox/modules/ServerDLL.cpp

```diff
@@ -206,15 +206,40 @@ void ServerDLL::Hook(const std::wstring& moduleName, HMODULE hModule, uintptr_t
 			break;
 
 		case 1:
+			off1M_bDucked = 2;
```

Comment:
Why did the `case 1:` offsets change?

Reply:
Maybe because you never added a pattern to go with those offsets in the first place, and so it was pointless to keep.